charmcraft/templates/init-simple/metadata.yaml.j2

```
@@ -1,25 +1,43 @@
-# Copyright {{ year }} {{ author }}
-# See LICENSE file for licensing details.
+# This file populates the Overview on Charmhub.
+# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
 
-# For a complete list of supported options, see:
-# https://juju.is/docs/sdk/metadata-reference
+# The charm package name, no spaces (required)
+# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
 name: {{ name }}
-display-name: |
-  TEMPLATE-TODO: fill out a display name for the Charmcraft store
+ 
+# The following metadata are human-readable and will be published prominently on Charmhub.
+
+# (Recommended)
+display-name: Charm Template
```

```suggestion
display-name: Display Name
```

I would even consider uppercase dash separation of name as useful alternative

---

@jnsgruk your call ^

---

Personally I think it was fine as is 🤷🏼‍♂️

---

I'll keep the original. Thanks!